plugins/modules/rabbitmq_plugin.py

```diff
@@ -29,6 +29,11 @@
       - Does not disable plugins that are not in the names list.
     type: bool
     default: "no"
+  node:
+    description:
+      - Erlang node name of the rabbit we wish to configure.
```

Comment:
```suggestion
      - Erlang node name of the rabbit we wish to configure.
```
Please also add a note that this use `-n` under the hood and since which version this is supported.
If users don't use this argument, everything should be OK in terms of backwards compatibility, right?